containers-tests/benchmarks/Set.hs

```diff
@@ -45,6 +45,8 @@ main = do
         , bench "alterF:four:strings" $ whnf (alterF_four strings) strings_s
         , bench "alterF_naive:four" $ whnf (alterF_naive_four elems) s
         , bench "alterF_naive:four:strings" $ whnf (alterF_naive_four strings) strings_s
+        , bench "powerSet" $ whnf S.powerSet (S.fromList[1..20])
```

Comment:
20 seems a bit small.

Reply:
I mean I want 21 instead of 20. Or even better, 19, 20, and 21. Just to bump things up to around a second. Seeing a few gives some sense of actual growth.